src/transformers/pipelines/base.py

```diff
@@ -772,6 +814,9 @@ def __init__(
         if self.framework == "pt" and device is not None:
             self.model = self.model.to(device=device)
 
+        if self.framework == "flax":
```

Comment:
Awaiting confirmation if we should jit by default or have an extra argument. 

Something like below, discussed in https://github.com/huggingface/transformers/pull/14356/#issuecomment-991028446 
>  I think we should delegate it to the user meaning that there will be both a jit=True/False and a parallel=True/False input that the user can specify. IMO both should default to False, but for the inference widget we should default jit to True I think (if JAX is used for large models)